dll/win32/winmm/time.c

```diff
@@ -462,6 +476,11 @@ DWORD WINAPI timeGetTime(void)
     if (pFnReleaseThunkLock) pFnReleaseThunkLock(&count);
     if (pFnRestoreThunkLock) pFnRestoreThunkLock(count);
 #endif
-
+	/* Use QPC if a high-resolution timer is requested (<= 5ms) */
+	if (TIME_qpcFreq.QuadPart != 0) {
+		QueryPerformanceCounter(&perfCount);
```

Comment:
normally ```QueryPerformanceCounter``` and ```QueryPerformanceFrequency``` won't fail.

Reply:
Only one check is actually needed (at line 430) since if QPF fails TIME_qpcFreq.QuadPart will remain 0 and thus this code will be skipped preventing the need for a second check.